region/maridia/outer/Fish Tank.json

```diff
@@ -2196,8 +2281,20 @@
       "id": 7,
       "name": "Insane Naked Jump Over Pirate (Left to Right)",
       "note": "This is particularly precise, and requires subpixel precision."
+    },
+    {
+      "id": 8,
+      "name": "Artificial Morph, Double Spring Ball Jump onto Frozen Fish",
```

Comment:
Its not crazy hard; the timing is somewhat lenient but I didn't see a good cue for it.   Resetting the room if you miss is the only really bad part to this, and setting up artificial morph again.